docs/pages/colors.mdx

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
These don't really belong in the "getting started" section, but might need to be in a separate section just for our design system tokens. Most design systems have this as "Foundations" which should fit here as well.